pyc_module.cpp

```diff
@@ -202,6 +202,23 @@ void PycModule::loadFromFile(const char* filename)
     m_code = LoadObject(&in, this).require_cast<PycCode>();
 }
 
+void PycModule::loadFromMarshalledFile(const char* filename, int major, int minor) {
+    PycFile in (filename);
+    if (!in.isOpen()) {
+        fprintf(stderr, "Error opening file %s\n", filename);
+        return;
+    }
+    PycMagic magic = version_to_magic(major, minor);
+    if (magic == PycMagic::INVALID) {
+        fprintf(stderr, "Unsupported version\n");
+        return;
+    }
+    setVersion(magic);
+    m_maj = major;
+    m_min = minor;
```

Comment:
`setVersion()` already sets these...  Additionally, since we're parsing the version directly, we may want to have a way to indicate that Unicode mode is being used for Python 1.6-2.7.  Perhaps checking for a 'u' suffix to the version number would be sufficient?

Reply:
Isn't the `CO_FUTURE_UNICODE_LITERALS` flag set within the code object itself, and wouldn't need to be explicitly added?

Reply:
The unicode bytecode flag is added when Python 2.x is run with the (undocumented) `-U` flag, and is not related to the `CO_FUTURE_UNICODE_LITERALS` code flag -- the two can be present or absent independently of each other...

However, from looking at the code, it looks like the only place we actually use `m_unicode` is when printing the Python version in the comments...  There may be something missing there that will need to be addressed eventually, but since this flag was rarely used and doesn't have any meaningful impact on the output, I suppose this can be ignored for now.